src/thunderbots/software/network_input/filter/robot_filter.h

```diff
@@ -40,9 +42,11 @@ class RobotFilter
     /**
      * Creates a new robot filter
      *
-     * @param id the id of the robot to filter
+     * @param current_robot_state the data of current state of the robot
+     * @param expiry_buffer_duration the time when the robot is determined to be out of
```

Comment:
Nit:

I would change this statement to do "...robot is determined to be removed from the field..." 
This would be clearer as 'out of the field' could be interpreted as outside of the bounds I think.

Reply:
comment changed